### ISSUE TITLE
Report ECT email address & CYA

### ISSUE THREAD

**Issue by mason-emily** [DFE-Digital/register-early-career-teachers#475]
## Description
This story will deliver the ability for schools to report an email address for a new ECT they are adding to their school, and then to view and check the details they have provided in the registration journey.

## User story

**As** a school user
**I need** to report the new ECT's email address and check the details I've entered
**So that** DfE have the correct details they need to enable ECT training and funding

## Prototype, screenshots & content
[Link to prototype for 'Add an ECT' journey ](https://ecf2-prototy-v3-oz6554dztlkwtv.herokuapp.com/v3/school/home/ects)
Password: review

[Link to 'What is [ECT name]'s email address?' page](https://ecf2-prototy-v3-oz6554dztlkwtv.herokuapp.com/v3/school/email-address)
Note, there are additional pages in the prototype between the email and CYA pages that are **not** in scope for this ticket.

Screenshots of relevant screens:
![Image](https://github.com/user-attachments/assets/16fb8291-fe11-4e06-8736-3a1ce708e8d9)

Content should be as per the screenshots above.
URL and validation messages should be as per [this doc](https://educationgovuk.sharepoint.com/:x:/r/sites/TeacherServices/Shared%20Documents/Teacher%20Continuing%20Professional%20Development/Teacher%20CPD%20Team/Register%20early%20career%20teachers/Beta/Design/Register%20ECTs%20page%20titles%20and%20URLs.xlsx?d=w0c1578ea643147b29357c7f610ee178f&csf=1&web=1&e=U8iIdi).

## Acceptance criteria

- [x] 1. When user clicks 'Confirm and continue' on the 'Check if this is your ECT' page (#473), they will be navigated to the 'What is [ECT name]'s email address?' page.
- [x] 2. The H1 question will pull in the ECT's name e.g. What is [John Smith]'s email address?
- [ ] ~~3. User can enter an email address. See [here](https://educationgovuk.sharepoint.com/:x:/r/sites/TeacherServices/Shared%20Documents/Teacher%20Continuing%20Professional%20Development/Teacher%20CPD%20Team/Register%20early%20career%20teachers/Beta/Design/Register%20ECTs%20page%20titles%20and%20URLs.xlsx?d=w0c1578ea643147b29357c7f610ee178f&csf=1&web=1&e=CHF69R&nav=MTVfezk0RjM1RTQ2LUM5MjMtNEQ3Ny1BREQ2LTQzNjY0Q0I1QjAzRn0) for validation rules and error message content for the email field.~~
- [x] 4. When user clicks 'Continue' on the 'What is [ECT name]'s email address?' page, they will be navigated to the 'Check your answers before submitting' page. Note, there are additional pages in the prototype that are not within scope for now.
- [x] 5. On the 'Check your answers before submitting' page, the following detail will be shown:
- [x] 5a. Name - first and last name from TRS record
- [x] 5b. TRN 
- [x] 5c. Date of birth
- [x] 5d. Email address - as entered on the 'What is [ECT name]'s email address?' page
Note, there are additional rows and 'Change' links in the prototype which are **not** in scope for this ticket.
- [x] 6. There will be a 'Back' link on each page that takes the user back to the previous screen in the journey.

## Linked tickets
Dependent on #473

**Comment by gracecbrennan** [comment]
@mason-emily is going to revisit ticket to consider if we're going to be including change functionality.

**Comment by claire-hughez** [comment]
We don't want to start this yet - some things left to think about for CYA

**Comment by gracecbrennan** [comment]
@tonyheadford to take a look

**Comment by mason-emily** [comment]
AC3 validation rules not implemented - @TobyRet @claire-hughez is it best to write the validation as a separate ticket?

**Comment by claire-hughez** [comment]
I don't mind, I think this depends what's easier for dev?
